FindOpenStudioSDK.cmake

```diff
@@ -1,78 +1,148 @@
 set(OPENSTUDIO_VERSION_MAJOR 3)
-set(OPENSTUDIO_VERSION_MINOR 0)
-set(OPENSTUDIO_VERSION_PATCH 1)
+set(OPENSTUDIO_VERSION_MINOR 1)
+set(OPENSTUDIO_VERSION_PATCH 0)
 set(OPENSTUDIO_VERSION "${OPENSTUDIO_VERSION_MAJOR}.${OPENSTUDIO_VERSION_MINOR}.${OPENSTUDIO_VERSION_PATCH}")
 
-find_package(openstudio "${OPENSTUDIO_VERSION}" CONFIG)
+#If this is a release enter the SHA as "+79857912c4"
+#set(OPENSTUDIO_VERSION_SHA "+09b7c8a554")
+#If this is a pre-release enter the pre-release and SHA as "-rc1+79857912c4"
+set(OPENSTUDIO_VERSION_SHA "-alpha+fa9a77bf17")
+
+# Paths where the cmake-downloaded archives will be put
+set(OPENSTUDIO_ARCHIVE_DIR "${PROJECT_BINARY_DIR}/OpenStudio-${OPENSTUDIO_VERSION}")
```

Comment:
I'm putting the archives in `build/OpenStudio-3.1.0 `for eg, instead of `build/`. That in order to be able to leverage caching a bit more. So far I haven't really found that caching the entire build dir works, but caching the OpenStudio tar.gz appears to. (perhaps it's because I cache a folder that is inside of another folder I'm trying to cache though... I didn't think of that ealier).